src/main/resources/config/liquibase/changelog/20190709161437_added_entity_OrganizationError.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="utf-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd
+                        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+
+
+    <!--
+        Added the entity OrganizationError.
+    -->
+    <changeSet id="20190709161437-1" author="jhipster">
+        <createTable tableName="organization_error">
+            <column name="id" type="uuid">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="entity_name" type="varchar(255)">
+                <constraints nullable="true" />
+            </column>
+
+            <column name="field_name" type="varchar(255)">
+                <constraints nullable="true" />
+            </column>
+
+            <column name="external_db_id" type="varchar(255)">
+                <constraints nullable="true" />
+            </column>
+
+            <column name="invalid_value" type="text">
+                <constraints nullable="true" />
+            </column>
+
+            <column name="cause" type="varchar(255)">
+                <constraints nullable="true" />
+            </column>
+
+            <column name="organization_id" type="uuid">
+                <constraints nullable="true" />
+            </column>
+
+            <column name="data_import_report_id" type="uuid">
+                <constraints nullable="true" />
+            </column>
+
+            <!-- jhipster-needle-liquibase-add-column - JHipster will add columns here, do not remove-->
+        </createTable>
+
+    </changeSet>
+
+    <changeSet id="20190709161437-1-relations" author="jhipster">
+
+    </changeSet>
+    <!-- jhipster-needle-liquibase-add-changeset - JHipster will add changesets here, do not remove-->
+
+    <!--
+        Load sample data generated with Faker.js
+        - This data can be easily edited using a CSV editor (or even MS Excel) and
+          is located in the 'src/main/resources/config/liquibase/data' directory
+        - By default this data is applied when running with the JHipster 'dev' profile.
+          This can be customized by adding or removing 'faker' in the 'spring.liquibase.contexts'
+          Spring Boot configuration key.
+    -->
+    <changeSet id="20190709161437-1-data" author="jhipster" context="faker">
```

Comment:
I think this is not required. This is added by newer version of Jhipster. We do not need this. I think you can remove this along with organization_error.csv.

Reply:
done